xdsl/dialects/experimental/math.py

```diff
@@ -95,6 +93,20 @@ class Atan2Op(IRDLOperation):
     rhs: Operand = operand_def(AnyFloat)
     result: OpResult = result_def(AnyFloat)
 
+    # TODO: use __init__
```

Comment:
?

Reply:
Sorry I forgot to remove the todo after changing that